Orange/widgets/widget.py

```diff
@@ -114,6 +114,10 @@ class OWWidget(QDialog, Report, metaclass=WidgetMetaClass):
     #: area to the right of the `controlArea`).
     want_main_area = True
     #: Should the widget construct a `controlArea`.
+    want_standard_buttons_box = "vertical"
```

Comment:
`want_standard_buttons_box` default is now `"auto"` (with `"horizontal"`, `"vertical"` and `None` as other options). `"auto"` will give vertical layout if there are more than two buttons, otherwise horizontal.